src/android/emulator/avd_list.rs

```diff
@@ -31,3 +31,9 @@ pub fn avd_list(env: &Env) -> Result<BTreeSet<Emulator>, Error> {
     })
     .map_err(Error::ListAvdsFailed)
 }
+
+fn is_emulator_log_line(name: &str) -> bool {
+    ["INFO    |", "WARNING |", "ERROR   |"]
```

Comment:
```suggestion
    ["INFO", "WARNING", "ERROR"]
```
just in case the tabulation changes between OS or some other formatting condition..

Reply:
I find that hard to change but if it ever does, we can always adapt later.

I included the space and `|`, to ensure that it is in fact a log line and not just weirdly named emulator, because you can name an emulator `INFO` but you can't have `|` in the name.

Maybe we can change the detection to check for `|` and that would be enough?

Reply:
just leave it as is then, they might also change the output layout in the future anyways